app/services/services.py

```
@@ -0,0 +1,81 @@
+
+def get_account_service(db: Annotated[AsyncSession, Depends(get_db)], apikey: str) -> AccountService:
+    account_repository = AccountRepository(db)
+    worlds_repository = WorldsRepository(db)
+    gw2_client = GW2Client(apikey)
+    return AccountService(account_repository, worlds_repository, gw2_client)
+
+
+@cache(expire=settings.CACHE_TTL_NORMAL_SECONDS, namespace="apikey", key_builder=cache_key_builder)
+async def validate_api_key(
+        authorization: str = Header(..., description="Authorization header: Bearer <API_KEY>"),
+        api_key_service: ApiKeyService = Depends(get_api_key_service)) -> dict:
```

Ruff (0.14.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)